CHANGELOG.md

```diff
@@ -6,16 +6,20 @@ High five!
 
 ## New features
 
-* Single element dimension definition, which can be used to 
+* Single element dimension definition, which can be used to
 fix the value of a dimension during optimization.
-* `total_iterations` function of `BayesSearchCV` that 
-counts total iterations needed to explore all subspaces. 
-* Add iteration event handler for `BayesSearchCV`, useful 
+* `total_iterations` function of `BayesSearchCV` that
```

Comment:
Oops it is not actually a function it is a `@property`

Reply:
Fixed